### ISSUE TITLE
omemo

### ISSUE THREAD

**Issue by jonasstein** [xdch47/gpo-xdch47#10]
https://github.com/xdch47/gpo-xdch47/blob/859e29554c61291c0536b8c3074e4225894efb14/net-im/gajim/gajim-1.8.0.ebuild#L62C1-L62C1

I think omemo is an unconditional part in 1.8.0

die is missing in https://github.com/xdch47/gpo-xdch47/blob/master/net-im/gajim/gajim-1.8.0.ebuild#L87

**Comment by xdch47** [comment]
hi,
that was what i thought, too - however it looks like gajim now depends on https://dev.gajim.org/gajim/omemo-dr.
e.g.: https://dev.gajim.org/gajim/gajim/-/blob/master/gajim/common/modules/omemo.py#L45

without omemo-dr i was not able to get omemo in gajim running.
thanks for the other remark, it tried to clean the "the experimental" ebuild up (129a610)

**Comment by jonasstein** [comment]
Yes, that is what I meant. 
Did you check, if gajim-1.8.0 works without omemo-dr? I did not check it, but I doubt that it is possible to use it without `USE=omemo`. If this is true, we can drop `omemo` and add it to the permanent dependencies.

**Comment by xdch47** [comment]
yes, thats true - i tried to update the dependencies here: https://github.com/xdch47/gpo-xdch47/blob/e58b64907f08f7d660eb808cc8466a5036240cb6/net-im/gajim/gajim-1.8.0-r1.ebuild
